src/background/responders.ts

```diff
@@ -130,5 +93,105 @@ export const responders: { [T in Action['type']]: Responder<T> } = {
       justTweeted: null,
       alert: 'POST TWEET FAILURE'
     }
+  },
+  'chrome.windows.getAll'(state, action) {
```

Comment:
we can replace the dots with underscores here

Reply:
I kinda like this as is - makes it easy to do a search for the chrome function being called and then the responder as well